colour.py

```diff
@@ -192,6 +192,8 @@
     (255, 255, 255): ['White']
 }
 
+COLOR_NAME_TO_RGB.update(XKCD_RGB_TO_COLOR_NAMES)
```

Comment:
Well, I can't really see how this even runs : `COLOR_NAME_TO_RGB` is not defined at this point, and is defined the next instruction. Please also notice that the format you use is closer to `RGB_TO_COLOR_NAME` but a simple update anyway will not make it : the key is meant to be 3-uple of RGB integers... not floats. And the update will overwrite previous names instead of appending them upon key collision which I expect to happen.

Reply:
Yes, that's totally correct, the line should be : 
`RGB_TO_COLOR_NAME.update(XKCD_RGB_TO_COLOR_NAMES)`
I messed the pull request as i originally wrote the code in another folder and then copied it manually in a newly cloned repository to make the pull request, i though i could manage it as it's only one line but apparently not :sweat_smile: 
I though it was working because it wasn't throwing errors but you're right, it's unclean and probably overwriting keys, i'll look into a better solution !